### PR TITLE
Flatten russia marriage abroad

### PR DESCRIPTION
**Pull request by elliotcm**
https://trello.com/c/1LVE07jc/274-marriage-abroad-russia-flatten-to-18-outcomes

### CODE REVIEW

**Comment by elliotcm** [comment]
@barrucadu Oops.  What did I do wrong in the expected outcomes file?

**Comment by barrucadu** [comment]
I'm not sure.  When I ran the regression tests on this branch (`RUN_REGRESSION_TESTS=marriage-abroad ruby test/regression/smart_answers_regression_test.rb`), all the missing files got regenerated.

**Comment by elliotcm** [comment]
@barrucadu Updated the ethiopia stuff

File: doc/flattening-outcomes.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Flattening Marriage Abroad outcomes for a country
 
 1. Put the country in the right section of `lib/data/marriage_abroad_data.yml` and `test/data/marriage-abroad-questions-and-responses.yml`
-2. `bundle exec rake marriage_abroad:flatten_outcomes[<your country>]`
+2. `bundle exec rake marriage_abroad:flatten_outcomes[<your country>,<civil_partnership or same_sex_marriage]` e.g. `marriage_abroad:flatten_outcomes[russia,same_sex_marriage]`
```
Comment (bevanloon) [inline doc/flattening-outcomes.md]: minor nit pick missing closing angle bracket
`<civil_partnership or same_sex_marriage]` -> `<civil_partnership or same_sex_marriage>]`

Comment (elliotcm) [reply doc/flattening-outcomes.md]: Fixed

**Pull request merged**